### DISCUSSION TITLE
Problem with GitHub Students Pack access

### DISCUSSION THREAD

**Discussion by duevvadim** [community/community#17705] [GitHub Education]
Hello there!

I cannot upload my document - Proof of Study certificate. There is only a button "Take a picture", and there is no button "Upload a picture" such as shown in the [instruction ](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/apply-for-a-student-developer-pack)

So, I can only take pictures from my front camera and it is in a low-quality. The system does not verify it and I get the automated denial email again and again. Thus, I cannot request the discount for students. What can be the problem? Can I send this document to a human?

Here is my Proof of Study document. Is it an appropriate format?
![IMG_20220510_103123](https://user-images.githubusercontent.com/103727753/167607585-ffcda364-b976-43a4-a58a-882c65b762b7.jpg)

Here is how the interface of the window for uploading documents look like:

![Screenshot_2022_05_10_15_29_03_478_com_microsoft_office_outlook](https://user-images.githubusercontent.com/103727753/167609186-0ebb4c9b-d273-4c61-90e0-2c2a9343961a.jpg)
![Screenshot_2022-05-10-15-25-36-615_com android chrome](https://user-images.githubusercontent.com/103727753/167609195-4a110ddd-d46c-4a01-9dce-7eab0925901b.jpg)
![Screenshot_2022-05-10-15-27-25-660_com android chrome](https://user-images.githubusercontent.com/103727753/167609207-c6731a87-d8b4-4bc8-94ff-1497a41649c8.jpg)

**Comment by Abhinavch011** [comment]
you have to click while showing your face so try to upload college id card.

**Reply by choudharyal** [reply]
Is there any solution for this problem ? I am also facing the same issue of upload option not available

**Reply by Abhinavch011** [reply]
If you have student id card then click photo while showing your id card and ur face same time and u will get access within 15 to 30 mins i did the same.

**Reply by P10swati** [reply]
With phone or laptop?

**Reply by Abhinavch011** [reply]
laptop

**Comment by mlarasusan** [comment]
Hi  👋 sorry for the frustration! Would you be able to apply from another phone camera?

Sometimes, for a number of reasons, our system needs additional levels of verification and asks that you take an image rather than upload one. Other suggestions might be to try applying while you are on campus, if able to.

**Comment by sharklayer** [comment]
As I said in other post, none of these solutions work, it seems the only want you to beg until they get bored and decide to approve. I've been applying for two weeks now, that's just insane and ridiculous...

**Comment by DevSrijit** [comment]
I just screen shotted my school portal.. It had my name and class and stuff. I didn't even need to submit an ID except that screenshot.

**Reply by DevSrijit** [reply]
I got verified soon enough, and a few days later I got the student developer pack.

**Reply by nurullahsaeem** [reply]
> I got verified soon enough, and a few days later I got the student developer pack.

How did you upload the screenshot?

**Reply by EPSM2022** [reply]
I see no way to upload a screenshot.  That would be a truly logical method.  Instead, I am trying to take potato-quality pics with my phone that get rejected.  :/

**Reply by vincenttjia** [reply]
> > I got verified soon enough, and a few days later I got the student developer pack.
> 
> How did you upload the screenshot?

This might seems stupid, But what I did was install OBS and start virtual camera. Then take the picture which I was approved, you know what they say "Modern problem require modern solution"

but right now am having other issue https://github.com/orgs/github-community/discussions/158

**Reply by vineet-codes256** [reply]
These tricks didn't work for me.

**Comment by nurullahsaeem** [comment]
Did you find a solution yet?

**Comment by EPSM2022** [comment]
I would love to know why a picture is the only way to make this happen.  A PDF seems to be much more reasonable and less painful for the reviewers. 

Rejected 3 times with 3 different explanations, using a different pic to address the issues each time.  It is as if they do not want people to participate.

**Comment by sharklayer** [comment]
They like to make us suffer. Last friday I tried to send different academic proofs, showing my face, hiding my face, holding the document and placing in a plane object to take the picture, all of them were rejected... I tried 17 times in a space of 24 hour that friday and got rejected automatically them all.
We are just a joke to them...

**Reply by sharklayer** [reply]
Today i tried 3 more times applying from campus college and still got rejected. Rejected 24 times now...

**Comment by EPSM2022** [comment]
I am planning on printing multiple docs showing my student status and sending a barrage of attempts tomorrow.  I think I will quit at attempt 10. Maybe they have a disgruntled lone employee who is sitting at the other end who gets some small pleasure in hitting the reject button.
________________________________
